pkg/cmd/cmd.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"context"
+	"os"
+
+	"github.com/bwplotka/mdox/pkg/clilog"
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/observatorium/obsctl/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+const (
+	logFormatLogfmt = "logfmt"
+	logFormatJson   = "json"
+	logFormatCLILog = "clilog"
+)
+
+var logLevel, logFormat string
+var logger log.Logger
```

Comment:
I am not too keen on having global variables, I think we can create wrappers around the Run functions and pass loggers in there.